vocabulary-draft/continuous-integration-pipeline-events.md

```diff
@@ -0,0 +1,28 @@
+# Continuous Integration Events
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+These events are related to **Continuous Integration(CI)** activities. CI usually include activities such as building, testing, packaging and releasing software artifacts. 
+
+The following events represent concrete Tasks that are associated with the execution of CI pipelines:
```

Comment:
Too late I remembered https://github.com/cdfoundation/sig-interoperability/blob/master/docs/vocabulary.md#mapping-of-terms. Do we want to link to it here in the definition or use it in any way?

Reply:
@m-linner-ericsson I think we should rely on the shared terms.. but they are not defined yet. I am a strong believer that we should move forward and adapt when these terms are defined, so we should keep a close eye on that. As soon as we don't promote this vocabulary as a finished product we should be ok with pivoting when needed.

Reply:
As some tools use workflow instead of pipeline I was wondering if we should state say `CI pipelines/workflows` to indicate that we talking about both?

Reply:
We could also provide a general comment somewhere in the draft linking to that vocabulary document and stating that "we currently use the term 'pipeline' to denote a pipeline/workflow/... and the term 'step' to denote a job/stage/step" or something similar

Reply:
@e-backmark-ericsson Yes, I've added this to the Vocabulary PR